src/custom_nodes/object_detection/README.md

```diff
@@ -0,0 +1,62 @@
+# Custom node for object detection models from OpenVINO Model Zoo
+
+This custom node analyses the response of models from OpenVINO Model Zoo. Based on the inference results and the original image,
+it generates a list of detected boxes for following object recognition models. 
+Each image in the output will be resized to the predefined target size to fit the next inference model in the 
+DAG pipeline.
+Additionally to the detected text boxes, in the two additional outputs are returned their coordinates with information about coordinates
```

Comment:
I think there's too many repetitions in this sentence - "their coordinates with information about coordinates" what does it mean? It's a little confusing.

Reply:
Done